### PR TITLE
Addded PropertySetterRemoved and PropertyGetterRemoved

### PR DESCRIPTION
**Pull request by MarkBennis**

### CODE REVIEW

File: ApiCheck/Comparer/PropertyComparer.cs [hunk 1]
```diff
@@ -20,11 +20,25 @@ public override IComparerResult Compare()
       }
       if ((ReferenceType.CanWrite && ReferenceType.SetMethod.IsPublic) != (NewType.CanWrite && NewType.SetMethod.IsPublic))
```
[the remark this answers is not in the document]
Comment (MarkBennis) [reply ApiCheck/Comparer/PropertyComparer.cs]: Done

File: ApiCheck/Comparer/PropertyComparer.cs [hunk 2]
```diff
@@ -20,11 +20,25 @@ public override IComparerResult Compare()
...
       {
-        ComparerResult.AddChangedFlag("Setter", ReferenceType.CanWrite, Severities.PropertySetterChanged);
+          if (ReferenceType.CanWrite && ReferenceType.SetMethod.IsPublic)
+          {
+              ComparerResult.AddRemovedItem(ResultContext.Property, ReferenceType.Name, Severities.PropertySetterRemoved);
+          }
+          else
+          {
+              ComparerResult.AddChangedFlag("Setter", ReferenceType.CanWrite, Severities.PropertySetterChanged);
```
[the remark this answers is not in the document]
Comment (MarkBennis) [reply ApiCheck/Comparer/PropertyComparer.cs]: Changed PropertySetterChanged to PropertySetterAdded

**Review by swiebertje — APPROVED** [review]

**Comment by swiebertje** [comment]
@PMudra I sent you a message on Gitter. There's an issue with the package deployment.

**Comment by PMudra** [comment]
Thanks for reminding me. I will have a look soon. I guess I have to renew the nuget token.

**Comment by PMudra** [comment]
A new version of the nuget package including these changes has been deployed now: https://www.nuget.org/packages/ApiCheck/1.1.91

**Pull request merged**